Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -1674,7 +1674,12 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     constexpr auto updatedEventDelay = 25_ms;
 
     // Fire the updated event after a small delay to coalesce relevant changes together.
-    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(updatedEventDelay.seconds() * NSEC_PER_SEC)), dispatch_get_main_queue(), makeBlockPtr([this, protectedThis = Ref { *this }, tab = Ref { tab }]() {
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(updatedEventDelay.seconds() * NSEC_PER_SEC)), dispatch_get_main_queue(), makeBlockPtr([this, protectedThis = Ref { *this }, tabIdentifier = tab.identifier()]() {
+        // Get the tab again, it might have closed since this was scheduled.
+        auto tab = getTab(tabIdentifier);
```

Comment:
Please use `RefPtr` here assuming `tab` is ref counted.

Reply:
It is `RefPtr` already as a result of `getTab`.